_episodes/08-exercise.md

```diff
@@ -34,23 +117,29 @@ the different bacteria are. It already has tests written for most functions.
 >    the badges.
 > 4. Create a virtual environment on your computer for the project, and install
 >    the project's requirements, so you can run the test suite locally.
-> 5. Currently, some of the tests for the repository fail. Work out why this is
+> 5. The current code is very outdated by now and you will see in a moment that
+>    it does not work with a standard contemporary python installation anymore.
```

Comment:
capitalise Python

Reply:
Oh, wow! Python itself capitalises it. I could have sworn that the brand name is lowercase.